packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulator.tsx

```diff
@@ -1,23 +1,30 @@
-import { PropTypes as MobxPropTypes } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { localize } from '@deriv/translations';
-import NumberSelector from 'App/Components/Form/number-selector.jsx';
-import Fieldset from 'App/Components/Form/fieldset.jsx';
-import { connect } from 'Stores/connect';
+import NumberSelector from '../../../../../../App/Components/Form/number-selector.jsx';
+import Fieldset from '../../../../../../App/Components/Form/fieldset.jsx';
+import { connect } from '../../../../../../Stores/connect';
 import { getGrowthRatePercentage, getTickSizeBarrierPercentage } from '@deriv/shared';
 import classNames from 'classnames';
+import { useStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types.js';
 
-const Accumulator = ({ accumulator_range_list, growth_rate, onChange, tick_size_barrier }) => {
+type TAccumulator = {
+    accumulator_range_list: ReturnType<typeof useStore>['trade']['accumulator_range_list'];
+    growth_rate: ReturnType<typeof useStore>['trade']['growth_rate'];
+    onChange: ReturnType<typeof useStore>['trade']['onChange'];
+    tick_size_barrier: ReturnType<typeof useStore>['trade']['tick_size_barrier'];
```

Comment:
@henry-deriv let's not rely on what useStore returns until we actually start using useStore in this file